package.json

```diff
@@ -38,14 +37,10 @@
     "@semantic-release/github": "7.2.3",
     "@semantic-release/npm": "7.1.3",
     "@semantic-release/release-notes-generator": "9.0.3",
-    "babel-cli": "6.26.0",
-    "babel-core": "6.26.3",
-    "babel-preset-es2015": "6.24.1",
-    "babel-preset-stage-0": "6.24.1",
-    "codecov": "3.7.1",
-    "jasmine": "2.8.0",
-    "jasmine-spec-reporter": "4.0.0",
-    "nyc": "14.1.1",
+    "c8": "10.1.2",
```

Comment:
Why switch from nyc to c8? I believe we use nyc across our repos?

Reply:
`nyc` doesn't support es6 modules. Also c8 uses the same istanbul-reporter and `.nycrc` file as `nyc`.